src/main/resources/templates/jenkins/empty/regularRuns/Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+// ARTEMIS: JenkinsPipeline
+
+pipeline {
+        options {
+            timeout(time: #jenkinsTimeout, unit: 'MINUTES')
+        }
+        agent {
+            docker {
+                image 'maven:3.3.3'
+                label 'docker'
```

Comment:
Shouldn't we use the same Docker Image as defined in `ContinuousIntegrationService.java`?

Reply:
We can do that yes